### PR TITLE
Pre-release version 2.0.0

### PR DESCRIPTION
**Pull request by simonepri**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
-'use strict';
-var test = require('ava');
-var osFilterObj = require('./');
+import test from 'ava';
+import osFilterObj from './';
```
Comment (kevva) [inline test.js]: `import m from '.'`

File: package.json [hunk 2]
```diff
@@ -26,6 +26,10 @@
     "os"
   ],
   "devDependencies": {
-    "ava": "^0.0.4"
+    "ava": "*",
+    "xo": "^0.18.2"
+  },
+  "dependencies": {
```
Comment (kevva) [inline package.json]: `dependencies` should be listed before `devDependencies`.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "os-filter-obj",
-  "version": "1.0.3",
+  "version": "2.0.0",
```
Comment (kevva) [inline package.json]: I'll do this manually.

File: index.js [hunk 4]
```diff
@@ -1,33 +1,38 @@
...
+	if (!arr || arr.length === 0) {
+		return [];
 	}
 
-	if (!arr || !arr.length) {
-		return [];
+	platform = platform || process.platform;
+	arch = arch || require('arch')();
+
+	if (opts && opts.keep) {
```
Comment (kevva) [inline index.js:13]: This isn't documented.

File: package.json [hunk 5]
```diff
@@ -26,6 +26,10 @@
     "os"
   ],
   "devDependencies": {
-    "ava": "^0.0.4"
+    "ava": "*",
+    "xo": "^0.18.2"
```
Comment (kevva) [inline package.json]: `"xo": "*"`

**Review by kevva — CHANGES_REQUESTED** [review]

File: test.js (hunk 1 above)
Comment (simonepri) [reply test.js]: Also
osFilterObj -> m ??

File: index.js (hunk 4 above)
Comment (simonepri) [reply index.js:13]: I was thinking of making an other PR for it, but you're right. Done 👍

File: test.js (hunk 1 above)
Comment (kevva) [reply test.js]: I use `m` in tests for all my modules to keep consistency.

Comment (simonepri) [reply test.js]: Makes sense. Done!

File: index.js (hunk 4 above)
Comment (simonepri) [reply index.js:13]: You can see the preview here:
https://github.com/simonepri/os-filter-obj

**Comment by simonepri** [comment]
@kevva any news?

**Comment by kevva** [comment]
Will take a look at it later today.

**Comment by simonepri** [comment]
Thank you 🎉 
If other changes are needed, just ask.

**Comment by kevva** [comment]
Decided to use `keep` as the default behaviour instead https://github.com/kevva/os-filter-obj/commit/1c15d395891c9dd03b274e6fcd825f7c54d2d50b.

**Pull request merged**